docs/configuration.md

```diff
@@ -19,6 +19,8 @@ backup:
     image: kloeckneri/pgdump-gcs:latest
   mysql: {}
 monitoring:
+  # append as an ENV variable "PROMETHEUS_PUSH_GATEWAY" to the backup cronjob
+  promPushGateway: ""
```

Comment:
probably it's not case sensitive and just works for now but what you wrote in https://github.com/kloeckner-i/db-operator/pull/51/files#diff-4bb9f0a337e0474b8eac4826f09f80dc81fc181bd3ef5f311010ff9c123526d8R57 ist `promPushGateWay`

Reply:
fixed to `promPushGateway` for all cases.